roachpb/data_test.go

```diff
@@ -158,6 +158,24 @@ func TestNextKey(t *testing.T) {
 	}
 }
 
+func TestIsPrev(t *testing.T) {
```

Comment:
IsPrev() is a test for keys and it was out of place in the test file. Just a little housekeeping.